### DISCUSSION TITLE
How can I get the latest x series using DICOM WEB?

### DISCUSSION THREAD

**Discussion by aleja-lopez** [dcm4che/dcm4chee-arc-light#3435] [Q&A]
How can I get the latest x series using dicom web. I tried using the limit parameter but it throws me the first x series, also I can nor find a similar parameter but only return the last x series.

**Comment by vrindanayak** [comment]
W.r.t. date time fields, series ordering can be done only by newest / oldest Performed Procedure Step Start Date Time
